src/index.js

```diff
@@ -707,6 +721,16 @@ class State extends EventEmitter {
 	}
 
 	/**
+	 * Send an arbitrary message to the named channel. The type of the message will be coerced to
+	 * a Max type in the Max application by mira.channel
+	 * @param {String} channel - The name of the mira.channel objects that should receive this message
+	 * @param {Number|String|Array<Number|String>|Object} message - the message to send
+	 */
+	sendMessageToChannel(channel, message) {
```

Comment:
As outlined here https://github.com/Cycling74/xebra-communicator/pull/4#pullrequestreview-20078136

do we wanna give the option of passing a callback fct to handle errors in order to avoid throwing errors? Maybe it's fine. Just wanted to double check with u.